README.md

```diff
@@ -399,14 +404,12 @@ services:
 <br/>
 
 <details><summary>✏ <b>Code Examples</b></summary>
-//docker-compose file
-```
-// Show global-setup conditional migration command
-//Show npm script
+
+```javascript
+  await npmCommandAsPromise(['db:migrate']);
```

Comment:
The reader won't understand where is this coming from, don't count on reading everything before. Give more context.

Put jest config, put comment above "// In the global setup we also install all the tables"